src/server_manager/web_app/app.ts

```diff
@@ -53,6 +53,17 @@ function isManualServer(testServer: server.Server): testServer is server.ManualS
   return !!(testServer as server.ManualServer).forget;
 }
 
+async function isAccessKeyPortEditable(server: server.Server): Promise<boolean> {
+  try {
+    // -1 always fails, making this idempotent
+    await server.setPortForNewAccessKeys(-1);
+    // unreachable unless something went HORRIBLY WRONG
+    return false;
+  } catch (e) {
+    return !e.isNetworkError() && e.response.code !== 404;
```

Comment:
I don't see `code` in the `Response`interface; I think it is `status`? Also, we may want to check `e.response.code === 400` (we may get 5XXs)?

Reply:
Done